packages/template-retail-react-app/app/theme/components/project/footer.js

```diff
@@ -62,9 +62,6 @@ export default {
                 background: 'whiteAlpha.500'
             }
         },
-        localeDropdownOption: {
-            color: 'black'
```

Comment:
Had to do delete this theme property, because now we have to use native `<option>`.. and not the chakra-enhanced version that can accept style / theme props.